index.html

```diff
@@ -325,32 +325,32 @@ <h3>Skills</h3>
                 </li>
             </ul>
         </div>
-    </section> 
+    </section>
     <section class="contact-me" id="contact">
             <h1 class="contact-me-text">Contact me</h1>
             <p class="contact-me-brief">
                 If you have an application you are intrested in developing, a feature that
                 you need built or a project that needs coding.
                 I'd love to help with it.
             </p>
-            <form 
+            <form
                 action="https://formspree.io/f/xeqnbaww"
                 method="POST">
                 <div class="input-field-box">
                     <input class="input-field" type="text" name="name" maxlength="30" placeholder="Type your name" required>
                     <input class="input-field" type="email" name="email" placeholder="Username@domain.ext" required>
-                    <textarea 
-                        class="write-message" 
-                        name="write message" 
+                    <textarea
+                        class="write-message"
+                        name="write message"
                         placeholder="Write your message here"
-                        maxlength="100" 
-                        rows="5" 
+                        maxlength="100"
+                        rows="5"
                         cols="30"
                         required>
-                    </textarea> 
```

Comment:
![Screenshot from 2022-07-12 18-57-01](https://user-images.githubusercontent.com/93039359/178508868-e6015f12-c913-44d7-b0ea-681e4dc1b761.png)
- [ ] You have made a lot of effort here but please add a label for each input item for the accessibility purpose :+1: